src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/CollectionModelBinder.cs

```diff
@@ -147,7 +150,14 @@ protected virtual object CreateEmptyCollection(Type targetType)
         /// <returns>An instance of <paramref name="targetType"/>.</returns>
         protected object CreateInstance(Type targetType)
         {
-            return Activator.CreateInstance(targetType);
+            if (_modelCreate == null)
+            {
+                _modelCreate = Expression.
+                    Lambda<Func<object>>(Expression.New(targetType)).
+                    Compile();
```

Comment:
It's not a rule but please follow the convention used in the rest of this repo: Place member access operators after the newlines.